Lib/glyphsLib/classes.py

```diff
@@ -3980,7 +3980,7 @@ def _brace_coordinates(self):
             return None
 
         if self.parent.parent.format_version > 2:
-            return self.attributes["coordinates"]  # Glyphs 3
+            return (int(v) for v in self.attributes["coordinates"])  # Glyphs 3
```

Comment:
why int and not float?

Reply:
Went with int instinctively since all the tutorials I've ever seen using the brace trick use int values. Turns out the GlyphsApp UI also coerces int's when you try to input commas or points.

Reply:
ok.. that's odd, because in fvar axes coordinates are fractional Fixed16Dot16 so I don't see why the editor has to force them to be rounded to integer. Anyway, if you say glyphs wants them to be int, we could at least do `round(float(v))` so we don't simply truncate the float to int maybe. Or whatever Glyphs.app does.

Reply:
> Turns out the GlyphsApp UI also coerces int's when you try to input commas or points

are you actually sure about that?
I just tried to create an intermediate (brace) layer in Glyphs 3.2 (3189) and it does let me type a float (e.g. 201.5) in the intermediate layer coordinates. I can see it saved back to disk as well. So it seems like floats are supported, as it should.

Reply:
`float` is the right one, because in the if branch within that same method you're touching that handles the Glyphs 2 variant we do `return [float(c) for c in coordinates.split(",")]`

Reply:
Hm, interesting. When I input a float with period I get "invalid value", when I input a "float" with a comma the layer name changes... I'm on latest GlyphsApp 3.2 (3189). Anyway, I concur, float is the right choice.

<img width="619" alt="Screenshot 2023-04-24 at 17 02 11" src="https://user-images.githubusercontent.com/1893577/234019933-0e4b3661-e54e-42fd-aa20-f7c2713e889b.png">
<img width="412" alt="Screenshot 2023-04-24 at 17 02 19" src="https://user-images.githubusercontent.com/1893577/234019953-76a09703-5ebb-4768-b0c5-aec2f2b59647.png">

Reply:
Do you happen to be on a locale where comma is the decimal separator?

Reply:
I am, but as shown above, the comma doesn't convert to a float either, it adds another supposed axis location, e.g. splitting the `{22, 200}` and it becomes `{22, 5, 200}` when inputting `22,5`, which is totally wrong (of GlyphsApp), too. Either way, I'm not to bothered with what GlyphsApp does as long as the glyphsLib conversion ends up consistent with consistent/usable data for what we see in the GlyphsApp interface. I've not spent more time, though, debugging what leads to (this particular file?) having some layer locations parsed as strings and some as int's.

Reply:
<img width="154" alt="Bildschirmfoto 2023-04-24 um 23 43 12" src="https://user-images.githubusercontent.com/174660/234123202-b3cbd122-5d88-4b22-91de-96a62fe82f9a.png">
In Glyphs 3, the numbers use localized decimal separators. This looks a bit confusing in this case. But is should be treated as a float internally and written out as float when writing Glyphs 3 files. But not when writing v2 files. When setting it up in Glyphs 2 it will write whatever you have typed. I need to dig up the Glyphs 2 code to see if it actually used it.

Reply:
Ah okay, on second very much closer look I see the space vs no-space. 

In any case, the PR now coerces float.